### PR TITLE
feat: new resize_img_proportional setting

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: release-notes/release-notes52.md [hunk 1]
```diff
@@ -53,9 +53,11 @@ For information on using the `TextSeeker` API, see: [{{site.productname}} APIs -
 
 ### Updated the colorinput component visual appearance --- updated color swatches in input fields
 
-### Missing resize_img_proportional setting
+### New proportional image resizing setting
```
Comment (lnewson) [inline release-notes/release-notes52.md]: This isn't new and I believe it actually also controls table resizing. I'd double check with @jhaines though.

Comment (tylerkelly13) [reply release-notes/release-notes52.md]: so have it just been fixed? revived?

Comment (lnewson) [reply release-notes/release-notes52.md]: Fixed, as it broke back in I think 4.7.x.

Comment (tylerkelly13) [reply release-notes/release-notes52.md]: release note removed

File: _includes/configuration/resize-img-proportional.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## resize_img_proportional
+
+When a user resizes an image in the editor, this option controls whether image is resized proportionally or freely. If set to:
+
+- `true` --- When users resize an image, the image will be resized proportionally (both dimensions of the image are resized by the same percentage of length).
```
Comment (EkimChau) [inline _includes/configuration/resize-img-proportional.md]: we should mention, with the true setting, holding down shift while resizing will change the behaviour to free-form
and when set to false, holding shift while resizing will get proportional behaviours

Comment (jhaines) [reply _includes/configuration/resize-img-proportional.md]: yeah this exactly

Comment (tylerkelly13) [reply _includes/configuration/resize-img-proportional.md]: Updated with shift behaviour for both true and false options

**Pull request merged**